torch/csrc/jit/passes/fold_conv_bn.cpp

```diff
@@ -389,6 +461,7 @@ graph(%self, %input, %conv, %batchnorm):
     h.analyze(m, pattern);
     h.transform();
   }
+  reverseBiasForConvIfNotDefined(m);
```

Comment:
Need to reverse undefined tensor for bias, which causes following mutation pass (e.g. quantization) to support undefined tensor. I feel this is a cleaner alternative :thinking: